spanner/api/SpannerTest/Tests.cs

```diff
@@ -133,13 +133,82 @@ public SpannerTests(SpannerFixture fixture)
                 if (!_fixture.s_initializedDatabase)
                 {
                     _fixture.s_initializedDatabase = true;
+                    DeleteStaleBackupsAndDatabases();
                     InitializeInstance();
                     InitializeDatabase();
                     InitializeBackup();
                 }
             }
         }
 
+        void DeleteStaleBackupsAndDatabases()
+        {
+            DatabaseAdminClient databaseAdminClient = DatabaseAdminClient.Create();
+            var instanceName = InstanceName.FromProjectInstance(_fixture.ProjectId, _fixture.InstanceId);
+            var databases = databaseAdminClient.ListDatabases(instanceName).Where(c => c.DatabaseName.DatabaseId.StartsWith("my-db-")).ToList();
+            var databasesToDelete = new List<string>();
+
+            // Delete all the databases created before 4 hrs.
+            var timestamp = DateTime.UtcNow.AddHours(-4).ToTimestamp().Seconds;

```

Comment:
This won't work as you think it will, again, you need something like TotalSeconds instead. Or better still if you just format the date and time. Or probably the database entity has a CreatedOn field, you can then create the database with a name from a Guid and use the CreatedOn filed to delete the older ones.